upstream-community-operators/kubevirt/kubevirtoperator.0.17.0-alpha.0.csv.yaml

```diff
@@ -0,0 +1,566 @@
+      mediatype: image/png
+  labels:
+    alm-owner-kubevirt: kubevirtoperator
+    operated-by: kubevirtoperator
+  selector:
+    matchLabels:
+      alm-owner-kubevirt: kubevirtoperator
+      operated-by: kubevirtoperator
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: true
+    - type: AllNamespaces
+      supported: true
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
```

Comment:
Should these be Cluster permissions, even in the SingleNamespace installMode?
OLM will upgrade namespaced permissions to clusterPermissions when an operator is installed into the AllNamespaces installMode

Reply:
I think we always need clusterPermissions (at least for most of the rules). The reason is, that while the operator might install KubeVirt in one namespace only, KubeVirt itself is a cluster wide add-on, which comes itself with several controllers and CRDs, and (at least at the moment) KubeVirt always watches for those CRDs on all namespaces. So, in order be able to create the ClusterRoles and -Bindings for the KubeVirt serviceaccounts, the operator itself also needs cluster wide permissions.

Reply:
If kubevirt ALWAYS watches all namespaces, then it should not support the SingleNamespace installMode. Please update the CSV accordingly

Reply:
Fixed: Set SingleNamespace to false

Reply:
@slintes ^^ something to fix on our side as well?

Reply:
I understand that with `SingleNamespace` installmode being `false`, `clusterPermissions` are fine. But we need to improve in case we are going to support `SingleNamespace` in future. @SamiSousa is that correct?